### ISSUE TITLE
Slow inference by espnet until restarting server

### ISSUE THREAD

**Issue by kamillle** [espnet/espnet#5562]
**Describe the bug**

After updating to 202310 from 0.10.6, slow inference by espnet 😢  It's resolved after restart a web server and I need to fast inference after initial startup.

**Basic environments:**
 - OS information: Ubuntu 20.04.6 LTS
 - python version: 3.8.10
 - espnet version: v202310
 - pytorch version: 2.1.0+cu121

We use FastAPI with espnet to serve as an inference server and every inferences are slow after initial startup. It's resolved by restarting server.

I need to any information related to this problem. thanks!

**Comment by sw005320** [comment]
Can you add more information?
What kind of configurations and models are used?

**Comment by kamillle** [comment]
@sw005320 

I have been repeatedly performing TTS inference using a Japanese VITS model with a GPU. 
When I used a random string of 30 Hiragana characters for TTS, 
there was no noticeable change in inference time despite repeated inferences. 

However, when I varied the number of characters randomly between 10 to 30, 
using random Hiragana for each TTS, 
I observed that the inference time gradually slowed down with each iteration. 

The average time for the first 10 inferences was about 100 milliseconds, 
but this increased to around 200 milliseconds after 1,000 repetitions.

Interestingly, when I switched from using the GPU for inference to using the CPU, 
this behavior did not occur.

Additionally, changing the PyTorch version from 2.1.0 to 1.13.1+cu117 seemed to 
resolve the issue even when using the GPU for inference. 
The inference time did not deteriorate under this configuration.

Given that this issue appears to be related to PyTorch, so I close this issue. thanks
